lib/puppet/provider/ini_setting/splunk.rb

```diff
@@ -13,7 +13,7 @@ def self.file_path
     File.join(@file_path, file_name)
   end
 
-  def self.set_file_path(path)
+  def self.set_file_path(path) # rubocop:disable Style/AccessorMethodName
```

Comment:
> 
```ruby
def self.file_path=(path)
```

leads to `Style/TrivialAccessors: Use attr_writer to define trivial writer methods.` which can be fixed by replacing the whole thing with 
```ruby
attr_writer :file_path
```
I think?

>
```ruby
Puppet::Type.type(res_type).provider(:ini_setting).file_path=(self[:server_confdir])
```
leads to `Style/SpaceAroundOperators: Surrounding space missing for operator =` at which point it was late and I got scared.